input/template.xml.in

```diff
@@ -1,24 +1,22 @@
  <facility>
+    <!-- {{ country }} -->
```

Comment:
I made a pull request regarding this issue, but it does it in a
maybe unnecessarily lengthy manner?
#14